plugins/editor/js/editor.js

```diff
@@ -1492,6 +1492,10 @@
                     $currentEditorToolbar.show();
                 });
 
+                $(document).on('click', 'input.CommentButton', function() {
+                    $currentEditorToolbar.show();
+                });
+
```

Comment:
Changing the click event above from

`$(document).on('click', 'a.WriteButton', function() {`
to
`$(document).on('click', 'a.WriteButton,input.CommentButton', function() {`

would have the same effect.
Also it saves one more handler to be attached to the document :D